unit-tests/utilities/concurrency/test-usage.cpp

```diff
@@ -0,0 +1,194 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#cmake:add-file ../../../src/dispatcher.cpp
+
+#include "../../test.h"
+#include <common/utilities/time/timer.h>
```

Comment:
I currently placed it under "Utilities".
I am not sure where to move this UT to. @maloel What do you think?

Reply:
Either here on in types/ is fine. Up to you.

Reply:
👍